### PR TITLE
ISPN-11539 Documentation edits and corrections

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11539

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/ref_hotrod_listeners.adoc [hunk 1]
```diff
@@ -511,10 +513,6 @@ content has been iterated over, events are received as normal, as cache
 updates are received.  If the cache is clustered, the entire cluster wide
 contents are iterated over.
 
-`includeCurrentState` also controls whether state is received when the node
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/ref_hotrod_listeners.adoc]: why was this sentence removed? 
AFAIK, it is true: https://github.com/infinispan/infinispan/blob/master/client/hotrod-client/src/main/java/org/infinispan/client/hotrod/annotation/ClientListener.java#L64

Comment (oraNod) [reply documentation/src/main/asciidoc/topics/ref_hotrod_listeners.adoc]: it seems like the following section "Listener Failure Handling" covers it in more detail so the sentence was redundant/repetitive.

**Comment by oraNod** [comment]
> In https://github.com/infinispan/infinispan/pull/8112/files#diff-ed196b2147d12b319df641f6df2a1f95R144,
> can you remove the incomplete sentence?

@pruivo I think I fixed it but not quite sure where you meant. could you please check?

File: documentation/src/main/asciidoc/topics/con_endpoints_memcached.adoc [hunk 2]
```diff
@@ -1,19 +1,27 @@
 = Memcached
-{brandname} provides an implementation of the Memcached text protocol for remote client access.
+{brandname} provides an implementation of the Memcached text protocol for
+remote client access.
 
-The {brandname} Memcached server supports clustering with replicated and
+[IMPORTANT]
+====
+The Memcached endpoint is deprecated and planned for removal in a future
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/con_endpoints_memcached.adoc]: is memcached supported in the product? if not, I would make this page community only.

File: documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc [hunk 3]
```diff
@@ -21,18 +21,10 @@ Check the server logs for the following messages:
...
-----
-
-Cluster shutdowns include the following messages:
-
-[source,options="nowrap",subs=attributes+]
-----
-ISPN080029: Cluster shutdown
-ISPN000080: Disconnecting JGroups channel cluster
+ISPN080003: {brandname} Server stopped
+ISPN080003: {brandname} Server stopping
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc]: duplicated?

File: documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc [hunk 4]
```diff
@@ -21,18 +21,10 @@ Check the server logs for the following messages:
...
+ISPN080003: {brandname} Server stopped
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc]: duplicated?

File: documentation/src/main/asciidoc/topics/proc_running_embedded.adoc [hunk 5]
```diff
@@ -11,8 +11,7 @@ Learn how to run {brandname} as an embedded data store in your project.
 GlobalConfigurationBuilder global = GlobalConfigurationBuilder.defaultClusteredBuilder();
 DefaultCacheManager cacheManager = new DefaultCacheManager(global.build());
 ConfigurationBuilder builder = new ConfigurationBuilder();
-builder.clustering().cacheMode(CacheMode.DIST_SYNC);
-cacheManager.administration().getOrCreate("myCache", builder.build());
+cacheManager.administration().withFlags(CacheContainerAdmin.AdminFlag.VOLATILE).getOrCreateCache("myCache", builder.build());
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_running_embedded.adoc]: `builder.clustering().cacheMode(CacheMode.DIST_SYNC);` deleted by mistake.

File: documentation/src/main/asciidoc/topics/ref_rest_caches.adoc [hunk 6]
```diff
@@ -234,7 +234,7 @@ Append the `extended` parameter to the query string to get additional informatio
 
 [source,options="nowrap",subs=attributes+]
 ----
-GET /cacheName/cacheKey?extended
+GET /{cacheName}/{cacheKey}/?extended
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/ref_rest_caches.adoc]: * are you sure there is a `/` before `?extended`?
* isn't missing `/rest/v2`?
```suggestion
GET /rest/v2/caches/{cacheName}/{cacheKey}?extended
```

File: documentation/src/main/asciidoc/topics/proc_starting_server.adoc [hunk 7]
```diff
@@ -25,7 +25,7 @@ Server logs display the following messages:
 ----
 ISPN080004: Protocol SINGLE_PORT listening on 127.0.0.1:11222
 
-ISPN080001: Infinispan Server started in 7453ms
+ISPN080001: {brandname} Server <$version> started in 4290ms
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_starting_server.adoc]: what do you think of removing the time? or use `<mm>ms`?

File: documentation/src/main/asciidoc/topics/con_endpoints_memcached.adoc (hunk 2 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/topics/con_endpoints_memcached.adoc]: @pruivo thanks I've made all the references to Memcached as community only

File: documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc [hunk 8]
```diff
@@ -21,18 +21,9 @@ Check the server logs for the following messages:
...
-ISPN080003: Infinispan Server stopped
-----
-
-Cluster shutdowns include the following messages:
-
-[source,options="nowrap",subs=attributes+]
-----
-ISPN080029: Cluster shutdown
-ISPN000080: Disconnecting JGroups channel cluster
+ISPN080003: {brandname} Server stopping
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc]: You forgot this one. There is a `server stopping` in line 24.

**Comment by pruivo** [comment]
integrated! thanks @oraNod !

**Pull request merged**